bftclient/test/bft_client_api_tests.cpp

```diff
@@ -362,7 +365,8 @@ TEST_F(ClientApiTestFixture, primary_gets_learned_on_successful_write_and_cleare
 TEST_F(ClientApiTestFixture, write_f_plus_one) {
   auto WriteBehavior = [&](const MsgFromClient& msg, IReceiver* client_receiver) {
     auto reply = replyFromRequest(msg);
-    client_receiver->onNewMessage((NodeNum)msg.destination.val, (const char*)reply.data(), reply.size());
+    client_receiver->onNewMessage(
+        (NodeNum)msg.destination.val, (const char*)reply.data(), reply.size(), (NodeNum)msg.destination.val);
```

Comment:
No need to pass the same value twice - let the default value take place.

Reply:
updated.